frontend/src/pages/Profile/ProfileAvatar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <CircleAvatar :avatarUrl="avatarUrl" radius="5" class="avatar-border" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import CircleAvatar from "@/components/CircleAvatar.vue";
+
+export default defineComponent({
+  props: {
+    avatarUrl: String,
+  },
+  components: {
+    CircleAvatar,
+  },
+});
+</script>
+
+<style scoped lang="less">
+// TODO: abstract this out with friend-cell's gradient border maybe,
+// but not sure how to abstract CSS
```

Comment:
wdym by this? You can define a gradient varibale in `index.less` and reuse it here

Reply:
can da gradient variable take in prop like radius

Reply:
Hmm, what's the radius for?

Reply:
the radius of the border is diff for FriendCell and ProfileAvatar

Reply:
:o which line is that referring to? ps not super familar with css haha